feature/CDH-5150-Events-on-completion (#3086)
SoftwareAG/c8y-docs: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md

The alarm will be active until it is cleared. The latter is the case when either an offloading run completes successfully, or the offloading configuration is deleted. Then, the active alarm is cleared, no matter if the alarms setting is activated or not. The alarm remains active if the offloading is unscheduled or raising alarms is deactivated. Again, the same applies to alarms being raised for compaction failures.

##### Raising events on completion {#raising-events}

> Sections [History per offloading pipeline](/datahub/working-with-datahub/#history-per-offloading-job) and [History of compactions per offloading pipeline](/datahub/working-with-datahub/#history-compaction-per-offloading-job) describe offloading and compaction job history respectively, both providing details for successful and failed runs. Additionally, an event can be raised within the {{< product-c8y-iot >}} platform when an offloading or compaction job has completed, either with success or failure. Under **Create events on** you can activate raising events for completed offloading and compaction runs. When activated and an offloading or compaction run is completed, a new event with execution details is raised. Per default, the setting is deactivated in order to prevent sending events unnecessarily. 

Reviewer: ```suggestion
The sections [History per offloading pipeline](/datahub/working-with-datahub/#history-per-offloading-job) and [History of compactions per offloading pipeline](/datahub/working-with-datahub/#history-compaction-per-offloading-job) describe the offloading and compaction job history respectively, both providing details for successful and failed runs. Additionally, an event can be raised within the {{< product-c8y-iot >}} platform when an offloading or compaction job has completed, either with success or failure. Under **Create events on** you can activate raising events for completed offloading and compaction runs. When activated and an offloading or compaction run is completed, a new event with execution details is raised. Per default, the setting is deactivated in order to prevent sending events unnecessarily. 
```

Author: Done

> The event is available in the {{< product-c8y-iot >}} [Device Management application](/device-management-application/monitoring-and-controlling-devices/#to-view-events). Each event comprises the standard event properties like time and type. The latter is named as *CDH_offloading_* followed by the uuid of the offloading pipeline. Additionally, each event comprises the name, id, execution id, and status of the associated offloading job. The status provides details of the offloading run like start and end time, failure/success, number of offloaded records.

Reviewer: ```suggestion
The event is available in the {{< product-c8y-iot >}} [Device Management application](/device-management-application/monitoring-and-controlling-devices/#to-view-events). Each event comprises the standard event properties like time and type. The latter is named as *CDH_offloading_* followed by the UUID of the offloading pipeline. Additionally, each event comprises the name, ID, execution ID, and status of the associated offloading job. The status provides details of the offloading run like start and end time, failure/success, number of offloaded records.
```

Author: Done

> The event is available in the {{< product-c8y-iot >}} [Device Management application](/device-management-application/monitoring-and-controlling-devices/#to-view-events). Each event comprises the standard event properties like time and type. The latter is named as *CDH_offloading_* followed by the UUID of the offloading pipeline. Additionally, each event comprises the name, ID, execution ID, and status of the associated offloading job. The status provides details of the offloading run like start and end time, failure/success, number of offloaded records.

Reviewer 2: [...] of the associated offoading **or compaction** job. [...]

Author: Done

> The option to send and receive such events allows you to act upon completion of an offloading run. Given the event type of the offloading you are interested in, e.g. CDH_offloading_92622259-c78b-408e-95e2-30944de2cc95, you can use the [Events REST API](http://cumulocity.com/api/core/#tag/Events) to retrieve associated events for that offloading pipeline. Alternatively, you can use the [Event notification API](https://cumulocity.com/api/core/#tag/Event-notification-API).

Reviewer: ```suggestion
The option to send and receive such events allows you to act upon completion of an offloading run. Given the event type of the offloading you are interested in, for example, CDH_offloading_92622259-c78b-408e-95e2-30944de2cc95, you can use the [Events REST API](http://cumulocity.com/api/core/#tag/Events) to retrieve associated events for that offloading pipeline. Alternatively, you can use the [Event notification API](https://cumulocity.com/api/core/#tag/Event-notification-API).
```

Author: Done
